src/orochi-network/orand-v1.md

```diff
@@ -0,0 +1,17 @@
+# Orand V1
+
+Orand project was built based on Elliptic Curve Verifiable Random Function (ECVRF), it's deterministic and verifiable based on EC assumption. Orochi Network unable to manipulate the result.
+
+To optimize operation cost and improve security we provided following features:
+
+- **Verifiable:** An Orand's epoch can be verified independently outside our system or can be verified by smart contract.
```

Comment:
```suggestion
- **Verifiable:** An Orand's epoch can be verified independently outside our system or can be verified by smart contracts.
```

Is it true that a single smart contract can verify the epoch, or any smart contract (I thought of this case. So I use ``smart contracts'')